Sources/ConcentricOnboarding/ConcentricOnboardingView.swift

```diff
@@ -70,13 +70,15 @@ public struct ConcentricOnboardingView : View {
     @State var progress: Double = 0
     @State var bgColor = Color.white
     @State var circleColor = Color.white
+    let nextIcon:String // the default icon is the cupertino next Icon button which is "chevron.forward but it can be change from the prop.
```

Comment:
"chevron.forward - missing a "
let nextIcon:String please insert a space between : and String